transcom/mymove
# [MB-16426] update prime simulator to include ability request sit address change request

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16426)

## Summary

This PR adds the ability to create a SIT address change request via prime simulator

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Login as a `prime` user
2. Access the `DSTSIT` move
3. Scroll to the service items section and click the `update service items` button
4. You should then be able to complete the form and verify the results in your DB and checking the move as a `TOO`

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![image](https://github.com/transcom/mymove/assets/111928238/81c84655-b1ee-431d-9ab7-c9481a0aa2ca)


![image](https://github.com/transcom/mymove/assets/111928238/e8efe20a-6833-4efd-a596-f11e9ba304ba)


![image](https://github.com/transcom/mymove/assets/111928238/69fd019f-881e-498c-a120-4a7bd702e405)

![image](https://github.com/transcom/mymove/assets/111928238/e608c408-1cd5-49c5-bafe-32ec98d352a3)
